### ISSUE TITLE
Dungeons Do Not Scale When Players Set Dungeon Scaling Profiles

### ISSUE THREAD

**Issue by LiskaQuicksilver** [rockfireredmoon/iceee#494]
This was reported to me some weeks ago by a player who attempted to scale various dungeons but the Scaling did not work.

I was able to duplicate this issue with Darsa and tried several different Scaling profiles to see if the dungeon (in this case the Corsica Lighthouse) would scale up according to the proper profile set. Not once did the dungeon scale to a more difficult setting. Unsure if this was due to this particular dungeon being on the Newbie Islands or if other factors were involved. The player seemed to insist it was happening with other instances as well.

As such, it's at least worth looking into if nothing else.

**Comment by rockfireredmoon** [comment]
Cannot reproduce. Dungeons seem to scale as expected. It is however possible this was fixed by an earlier problem I found in creature updates. We should also test parties, although I see no reason they should be broken.

While investigating this, I changed the Dungeon Scaling UI slightly so there are fewer clicks. Selecting a profile can be done by double clicking, a single click simply shows the details. Any action that selections will close the window.

I have also added the option to use spawn points (i.e. prop ones, such as a Lever) to be placed outside dungeon entrances, and be used to open up the scaler dialog. This makes the feature more discoverable. 

I haven't actually added to any entrances yet (other than locally in testing), will wait until i've spoken to @LiskaQuicksilver and @Heathendel and demonstrated this new feature.  If we are happy I'll remove the dungeon scale from Mod Options -> Dungeon Scaling.
